doc/classes/ProjectSettings.xml

```diff
@@ -1593,6 +1593,9 @@
 		<member name="rendering/textures/vram_compression/import_s3tc" type="bool" setter="" getter="" default="true">
 			If [code]true[/code], the texture importer will import VRAM-compressed textures using the S3 Texture Compression algorithm. This algorithm is only supported on desktop platforms and consoles.
 		</member>
+		<member name="rendering/vr/enabled" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], VR support is enabled in Godot, this ensures required shaders are compiled.
```

Comment:
Is it VR or XR?

Reply:
This change comes from #28207, basically this enabled stereoscopic rendering which I guess has a purpose for both AR and VR. Possibly renaming this setting to XR might be more future proof.